routes/api.php

```diff
@@ -14,6 +18,23 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::post('/tokens/authenticate', function (LoginRequest $request) {
+    $request->authenticate();
+
+    $user = User::where('email', $request->get('email'))->first();
+    $token = $user->createToken($user->username)->plainTextToken;
+
+    return response()->json([
+        'token' => $token,
+        'user_id' => $user->id
+    ]);
+});
+
+Route::middleware('auth:sanctum')->group(function() {
```

Comment:
This routes are more or less wrong. You should check any online good practice tutorial for that. It should be something like:

```php
    Route::get('/notes', [NoteController::class, 'getNotesByUsername']); // you should here user based on token not by what is in the URL
    Route::get('/notes/{noteId}', [NoteController::class, 'getNoteById']);
    Route::post('/notes', [NoteController::class, 'store']);
    Route::patch('/note/{noteId}', [NoteController::class, 'updateById']);
    Route::delete('/note/{noteId}', [NoteController::class, 'destroyById']);

    Route::get('/categories', [CategoryController::class, 'list']);
```